app/src/main/java/org/hackillinois/android/database/dao/EventDao.kt

```diff
@@ -15,6 +15,9 @@ interface EventDao {
     @Query("SELECT * FROM events WHERE startTime <= :time AND endTime >= :time")
     fun getAllEventsHappeningAtTime(time: Long): LiveData<List<Event>>
 
+    @Query("SELECT * FROM events WHERE startTime >= :startTime AND startTime < :endTime")
```

Comment:
We can sort this in here by doing: `... startTime < :endTime ORDER BY :startTime`